supplementary_style_guide/style_guidelines/general-formatting.adoc

```diff
@@ -10,8 +10,52 @@ Follow _The IBM Style Guide_  guidance of _day Month year_ for date formats, for
 When the format _day Month year_ causes a presentation or clarity issue, use instead: _Month day, year_ (for example, October 3, 2019).
 
 
-[[user-replaced-variables]]
-== User-replaced variables
+[[user-replaced-values]]
+== User-replaced values
+
+A _user-replaced value_, or _replaceable value_, is a value that the user must replace with a value that is relevant for their situation. User-replaced values are often found in places such as code blocks, file paths, and commands.
+
+Use descriptive names for user-replaced values and follow this general format: _<value_name>_.
+
+Ensure that user-replaced values have the following characteristics:
+
+* Surrounded by angle brackets (`< >`)
+* Separated by underscores (`_`) for multi-word values
+* Lowercase, unless the rest of the related text is uppercase or another capitalization scheme.
+* Italicized
+* If the user-replaced value is referencing a value in code or in a command that is normally monospace, also use monospace for the user-replaced value.
+
+.Example AsciiDoc: User-replaced value in a paragraph
+
+----
+Create an Ansible inventory file that is named `/_<path>_/inventory/hosts`.
+----
+
+This renders as follows:
+
+====
+Create an Ansible inventory file that is named `/_<path>_/inventory/hosts`.
+====
+
+To italicize a user-replaced value in a code block, you must add `subs="+quotes"` to the attribute list of the code block.
```

Comment:
@bergerhoffer I'm not an expert on the code block macros, but are you positive this is a requirement? Here's an example from our docs:
```
[options="nowrap" subs="normal"]
----
Host fully qualified DNS name of this server [_autodetected host name_]:
----
```
And I _think_ that when you have additional underscores in the variable, you should (must?) use double-underscores for italicizing, e.g.: __multi_word_variable__

If I'm wrong, I'd love to learn more.

Reply:
@stoobie In order for the italics to render, you need to add something to the attribute list to tell it to do so. `quotes` is the attribute that performs text styling substitutions. It looks like your example uses the `normal` attribute, which looks like it adds text styling and all other substitutions except callouts [1]. So yes the `normal` one seems to work fine too, but I'm not sure if we'd want to use that one as the suggestion, as it performs additional substitutions that might not be wanted. Let me know if you do think we should mention it as an alternative though.

[1] https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#attributes-and-substitutions

As for the double underscores, you definitely don't have to use them if the word has underscores, but you are right that there are some situations where the double underscores might be necessary. I can't remember exactly what those are - I think it might be when there is some additional formatting. However, I don't see any harm in using the double underscores in our example. I'll update.

Reply:
@bergerhoffer It sounds like you have a better grasp on the code block macros than I do. However, from what you wrote, it's not accurate to say that you must add `subs="+quotes"` if there are other alternatives. I think it would be nice to say that there are alternatives, and include a link to the quick reference you included here.

Reply:
@stoobie That page doesn't have a great anchor for that attribute section, so I don't necessarily want to link to that. But I've updated to include both quotes and normal as examples of attributes that apply text formatting. Let me know what you think!